code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -131,5 +133,5 @@
 
 /obj/item/storage/lockbox/t4/New()
 	..()
-	for(var/i = 0, i < 3, i++)
+	for(var/i in 0 to 3)

```

Comment:
odd little thing to include but hey

Reply:
This will make 4 of them; there's only supposed to be 3.